packages/vite-plugin-cloudflare/playground/dev-vars/src/index.ts

```diff
@@ -0,0 +1,10 @@
+export default {
+	async fetch(_req, env) {
+		const { ENV_NAME, MY_DEV_VAR_A, MY_DEV_VAR_B, MY_DEV_VAR_C } = env;
+		const dotDevDotVarsVariables = { MY_DEV_VAR_A, MY_DEV_VAR_B, MY_DEV_VAR_C };
```

Comment:
```suggestion
		const { ENV_NAME, ...dotDevDotVarsVariables } = env;
```

Reply:
I purposely didn't use the rest operation to be more explicit here

my version is I think also more future proof since later we can expand this app (adding other bindings for example) without breaking this test

I'm happy to go with the proposed code anyways (neither make a huge difference in my opinion) 👍